pkg/vsphere/toolbox/backdoor.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
```

Comment:
This is linux specific given the vmw-guestinfo package - should it be backdoor_linux.go?